.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ env:
   # Once 1.76 coes out, we can use that instead of nightly.
   DOCS_RUST_VERSION: "nightly-2023-12-17"
   # This is the MSRV used by `wgpu` itself and all surrounding infrastructure.
-  REPO_MSRV: "1.76"
+  REPO_MSRV: "1.79"
```

Comment:
please also update the rust toolchain file and other mentions of that version

Reply:
i had missed it in the root cargo.toml. everywhere else is already done, other mentions of the version relate to `CORE_MSRV`. not sure if FF would be fine with bumping to 1.79